test/integration/connect/envoy/main_test.go

```diff
@@ -22,6 +23,10 @@ var (
 func TestEnvoy(t *testing.T) {
 	flag.Parse()
 
+	if *flagWin == true {
+		travel_dir("../../../")
```

Comment:
I feel that this argument is a bit strange. Perhaps separating it into a variable might be clearer.